app/main/views/templates.py

```diff
@@ -866,7 +866,7 @@ def delete_service_template(service_id, template_id):
     try:
         last_used_notification = template_statistics_client.get_last_used_date_for_template(service_id, template.id)
         message = (
-            "This template has never been used."
+            "This template hasn't been used within last year."
```

Comment:
GOV.UK style guide [says](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#contractions):
> Avoid negative contractions like can’t and don’t. Many users find them harder to read, or misread them as the opposite of what they say. Use cannot, instead of can’t.

So:
```suggestion
            "This template has not been used within last year."
```